SPECS/vim/vim.spec

```diff
@@ -66,117 +63,126 @@ au BufEnter,BufNew *.py set tabstop=4 shiftwidth=4 expandtab
 if ! isdirectory("~/.vim/swap/")
         call system('install -d -m 700 ~/.vim/swap')
 endif
-set directory=~/.vim/swap//
+set directory=~/.vim/swap/
 " End %{_sysconfdir}/vimrc
 EOF
 
 %check
 sed -i '/source test_recover.vim/d' src/testdir/test_alot.vim
-make test
```

Comment:
Could you add a short comment explaining what we're doing with this changes? I'm guessing we're disabling some invalid/irrelevant tests.

Reply:
Changes were a cherry-pick from 1.0-dev's vim. @chalamalasetty what was the reasoning behind this change?